src/wire/reconnection.ts

```diff
@@ -22,9 +22,9 @@ export class ExponentialReconnectionPolicy implements IReconnectionPolicy {
     private retries: number = 0;
 
     /**
-   * @param {Number} maxDelay maximum duration to wait between reconnection attempts
-   * @param {Number} baseDelay delay, in milliseconds, to use in
-   */
```

Comment:
hah so glad they fixed this bug, it was super annoying. Btw we should update prettier settings here to match the rest of our projects